client/src/modules/StudentDashboard/components/MainStatsCard.tsx

```diff
@@ -1,54 +1,113 @@
 import * as React from 'react';
-import { TrophyOutlined } from '@ant-design/icons';
-import { Col, Row, Typography } from 'antd';
+import { TrophyOutlined, FireOutlined } from '@ant-design/icons';
+import { Card, Typography } from 'antd';
 import dynamic from 'next/dynamic';
-import CommonCard from './CommonDashboardCard';
 
 const PerformanceChart = dynamic(() => import('./PerformanceChart'), { ssr: false });
 
+const { Title, Text } = Typography;
+const PERCENT_TO_GET_CERTIFICATE = 70;
```

Comment:
where did you get this number?

Reply:
in [issue#1375](https://github.com/rolling-scopes/rsschool-app/issues/1375) it's used on figma layout. Maybe it would have be better to have some app-level constant for that, is there any I should use?

Reply:
Sorry, I think it's better to remove right now. % and condition to get certificate are unique per course. There are no any formal rules right now and 70% could lead to confusion for example for ios courses or others.

Reply:
done